### ISSUE TITLE
[SOLVED] Theme/template/CSS not loading or missing

### ISSUE THREAD

**Issue by jshrek** [freescout-helpdesk/freescout#1886]
So I moved my freescout install to a different subdomain on my server from subdomain.mydomain.com to newsub.mydomain.com

I changed the APP_URL in the .env file and then I ran `php artisan freescout:clear-cache` which was succesful:
`The compiled services & packages files have been removed.
Cache cleared successfully.
Compiled views cleared!
Configuration cache cleared!
Configuration cached successfully!
Created: storage/app/public/js/vars.js`

However I am now seeing plain text layout of all screens (even login) as if there is no css or the css is not loading (see screenshots below).

Did I miss a setting/change somewhere or is this a bug?

![screenshot1](https://user-images.githubusercontent.com/19673842/161791030-0b3fb306-fbf3-4ad6-a42f-3c02ef151d89.jpg)
![screenshot](https://user-images.githubusercontent.com/19673842/161791039-857f13aa-fee6-4bd9-841c-0b6b6339e3d8.jpg)

Note as a further investigation, if open the console for developers I see the error (see attached screenshot):
`Uncaught ReferenceError: initWorkflowsBulk is not defined`

![screenshot3](https://user-images.githubusercontent.com/19673842/161792000-498a6522-a602-42cb-8105-54f6e10b09a4.jpg)

EDIT:
I also clicked on System > Tools > Clear Cache but this did not resolve issue.
Also, I am running v1.8.4 ... it says there is an update available v1.8.10 but when I click on UPDATE NOW, nothing happens.

EDIT:
I ran update from command line  (something like php artisan freescout:update) and it is now updated to v1.8.10 but this did resolve the issue.

EDIT:
I found the following in my logs:
```
[2022-04-05 14:30:19] production.ERROR: Error: File '/home/something/public_freescout/public/modules/tags/css/module.css' does not exist; File: /home/something/public_freescout/overrides/devfactory/minify/src/Providers/BaseProvider.php (124)  
[2022-04-05 14:30:19] production.ERROR: Error: File '/home/something/public_freescout/public/modules/tags/js/laroute.js' does not exist; File: /home/something/public_freescout/overrides/devfactory/minify/src/Providers/BaseProvider.php (124)
```
And when I check the modules directory, I am not able to access any of the module folders... its like they dont exist.
I tried de-activating the modules, but the De-Activate buttons don't work (probably javascript related). 
I tried deleting the folders in the modules directory, but the modules still show as Active... How can I de-activate the modules if the deactiate button does not work?

**Comment by jbadson** [comment]
This can happen either because your APP_URL is not correct or because your caches are broken.

First, make sure your new APP_URL is correct and reachable from the browser.
Second, try the following to clear and rebuild caches:
`php artisan freescout:clear-cache`
`php artisan freescout:build`
`php artisan freescout:module-build`
Then make sure you don't have any permissions issues
`sudo chown -R www-data:www-data /var/www/html` , substituting the user, group, and path with appropriate values for your installation.

**Comment by jshrek** [comment]
Ok so I think the issue is that the modules directory uses symlinks, so when you "move" your installation llike I did, those symlinks break and it can cause problems like I was seeing above.

What I did was delete all the folders in the public/modules directory that were no longer accessible (which was all of them for me). But even after doing this, when I went to MANAGE > MODULES it still showed all the as ACTIVE (and the Deactivate button would not work).

The second thing I did was run the following command from the command line:
`php artisan freescout:module-install`
This asked me if I wanted to install all available modules and I said YES to each one and it said it was creating symlink for each and boom ... problem solved!
